File: 2025/J/answers/java/Solution.java
import java.util.*;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        if (!sc.hasNext()) return;
        String s = sc.next();
        
        long score = 0;
        int n = s.length();
        for (int i = 0; i < n; i++) {
            if (s.startsWith("ha", i)) score += 1;
            if (s.startsWith("boooo", i)) score -= 1;
            if (s.startsWith("bravo", i)) score += 3;
        }
        System.out.println(score);
    }
}

File: 2025/J/answers/kt/Solution.kt
import java.util.*

fun main() {
    val sc = Scanner(System.`in`)
    if (!sc.hasNext()) return
    val s = sc.next()

    var score = 0L
    val n = s.length
    for (i in 0 until n) {
        if (s.startsWith("ha", i)) score += 1
        if (s.startsWith("boooo", i)) score -= 1
        if (s.startsWith("bravo", i)) score += 3
    }
    println(score)
}
